### PR TITLE
[DJFS-95] Loops and Numbers

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Loops and Numbers
- `Created two if statements that looped through and produced numbers from 100 to 150 and even numbers between 0 and 50.

### CODE REVIEW

File: submissions/chabuChabu/php/LoopsAndNumber.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+//Display every number between 100 and 150
+
+for($i = 100;$i < 151;$i++){
```
Comment (adaugochi) [inline submissions/chabuChabu/php/LoopsAndNumber.php:5]: The question says `between 100 and 150`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/php/LoopsAndNumber.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ echo $i."<br>";
+}
+
+//Display even numbers between 0 and 50
+
+$x = 0;
+
+while($x < 50) {
+    $x += 2;
```
Comment (adaugochi) [inline submissions/chabuChabu/php/LoopsAndNumber.php:14]: This approach is ok, but not a good way of knowing if a number is an even number or not. You will have to do a rework

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/php/LoopsAndNumber.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+//Display every number between 100 and 150
+
```
Comment (TundeOladejo) [inline submissions/chabuChabu/php/LoopsAndNumber.php:4]: Remove extra space

File: submissions/chabuChabu/php/LoopsAndNumber.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+for($i = 101;$i < 150;$i++){
+ echo $i."<br>";
+}
+
+//Display even numbers between 0 and 50
+
```
Comment (TundeOladejo) [inline submissions/chabuChabu/php/LoopsAndNumber.php:10]: Remove extra space

File: submissions/chabuChabu/php/LoopsAndNumber.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+//Display every number between 100 and 150
+
+for($i = 101;$i < 150;$i++){
```
Comment (TundeOladejo) [inline submissions/chabuChabu/php/LoopsAndNumber.php:5]: Well done @ChabuChabu. Your for loop argument is hard to read. Try adding some spaces in between.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/php/LoopsAndNumber.php (hunk 3 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/php/LoopsAndNumber.php]: @TundeOladejo spaces removed

File: submissions/chabuChabu/php/LoopsAndNumber.php (hunk 5 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/php/LoopsAndNumber.php]: @TundeOladejo  for loop made more readable

**Review by TundeOladejo — APPROVED** [review]

**Pull request merged**